spec/controllers/nih_tables_controller_spec.rb

```diff
@@ -38,6 +38,37 @@
       expect(Zip::File.open_buffer(response.body).entries.map(&:name)).to include("Instructions.txt")
     end
 
+    it "should contain the samples & results table" do
+      get :show, params: { id: @samples_report.id }
+
+      expect(Zip::File.open_buffer(response.body).entries.map(&:name)).to include("#{@samples_report.name}_samples.csv")
+      expect(Zip::File.open_buffer(response.body).entries.map(&:name)).to include("#{@samples_report.name}_results.csv")
+    end
+
+    it "should contain the samples table with the correct data" do
+      get :show, params: { id: @samples_report.id }
+
+      samples_table = CSV.parse(Zip::File.open_buffer(response.body).entries.find{|e| e.name == "Test_samples.csv"}.get_input_stream.read, headers: true)
+      
+      #expect the samples table to contain the same number of rows as the samples report
```

Comment:
No need to comment every expectation. The expectations are enough to understand what's being tested.